### ISSUE TITLE
The background of the tab bar disappears

### ISSUE THREAD

**Issue by mrliptontea** [equinusocio/material-theme#192]
Hello there,

This is not a very big issue, but it's quite annoying. I use my own colour scheme with a little bit darker background in the editor.

When I start the Sublime tab bar's background is invisible and appears when I mouse over and out it. It disappears again if I switch to a file with different background colour, like [.todo](https://packagecontrol.io/packages/PlainTasks) or markdown files.

![st3-invisible-tabbar-2](https://cloud.githubusercontent.com/assets/3495587/8272403/44714254-1841-11e5-8ad4-5bd80712d11e.gif)

**Comment by equinusocio** [comment]
This happen with other themes?

**Comment by mrliptontea** [comment]
Nope, never before.
If it helps, some of theme I used are: [Afterglow], [Flatgrammer], [Flatland], [Predawn].

[Afterglow]: https://packagecontrol.io/packages/Theme%20-%20Afterglow
[Flatgrammer]: https://packagecontrol.io/packages/Theme%20-%20Flatgrammer
[Flatland]: https://packagecontrol.io/packages/Theme%20-%20Flatland
[Predawn]: https://packagecontrol.io/packages/Predawn

**Comment by equinusocio** [comment]
try disabling all packages and restart sublime

**Comment by mrliptontea** [comment]
I have just downloaded [Windows x64 build 3083](http://www.sublimetext.com/3). Folder `Data` is by default empty.

I have copied theme to `Data/Installed Packages/Material Theme.sublime-package` and have created a new file for user settings under `Data/Packages/User/Preferences.sublime-settings`, so these are the only packages.

Unfortunately, this is still happening.

![sublime fresh instance](https://cloud.githubusercontent.com/assets/3495587/8272617/ad655468-184a-11e5-8b83-8b1b55ea39f9.png)

**Comment by equinusocio** [comment]
Why you don't install the package by Package Control? You use the latest release 1.0.9?

**Comment by mrliptontea** [comment]
Well, I took the fresh editor, copied theme from my usual Sublime, where I installed it via Package Control. So it is the latest release, 1.0.9.

But wait, I've just git cloned the newest version from the master branch and it seems to be fixed! :smile:
I can confirm that if I checkout the `v1.0.9` tag (or any other release) it is broken, and it'fine on current master 050a98a

So it looks like I just have to wait for the next release.
